snap/snapcraft.yaml

```diff
@@ -0,0 +1,94 @@
+name: jitsi
+version: "1.0"
+summary: Jitsi
+description: |
+  This is a snap of Jitsi
+
+grade: stable
+confinement: strict
+base: core18
+
+# might need certificate configuration
+
+parts:
+  jicofo:
+    plugin: dump
+    source: https://download.jitsi.org/jitsi/debian/jicofo_1.0-508-1_amd64.deb
```

Comment:
What is the upgrade path here?

Reply:
This was made as an example using pre-built deb packages. You can use your own sources, like github repos, local files, etc. New versions would then reference newer sources.